jdaviz/configs/default/plugins/virtual_observatory/vo_plugin.py

```diff
@@ -48,7 +48,8 @@ class VoPlugin(PluginTemplateMixin, AddResultsMixin, TableMixin):
     source = Unicode("").tag(sync=True)
     coordframes = List([]).tag(sync=True)
     coordframe_selected = Unicode("icrs").tag(sync=True)
-    radius_deg = Float(1).tag(sync=True)
+    radius_val = Float(1).tag(sync=True)
+    radius_unit = Unicode("deg").tag(sync=True)
```

Comment:
can this make use of `UnitSelectPluginComponent`?

Reply:
Is this different than the global unit selection that currently only has solid angle implemented that we discussed at tagup?

Reply:
yes, it has an API and UI component for unit dropdowns

Reply:
Implemented in https://github.com/spacetelescope/jdaviz/pull/2872/commits/5742480b4ce43c0dbb6f5ab041929647b5be1982. Thanks for the tip!